src/rubicon/objc/api.py

```diff
@@ -246,28 +246,34 @@ def __repr__(self):
         return f"{type(self).__qualname__}({self.name_start!r})"
 
     def __call__(self, receiver, first_arg=_sentinel, **kwargs):
-        if first_arg is ObjCPartialMethod._sentinel:
+        # Ignore parts of argument names after "__".
+        order = [argname.split("__")[0] for argname in kwargs]
+        args = [arg for arg in kwargs.values()]
+
+        if first_arg is self._sentinel:
             if kwargs:
+                # This will be deleted when #26 is introduced.
```

Comment:
No need to flag this; #26 might not happen unless we can find a good implementation, and if that happens, the fact this isn't needed will become obvious.
```suggestion
```